### PR TITLE
Only allow machine users to call the registration endpoint

### PR DESCRIPTION
**Pull request by alexwlchan**

### CODE REVIEW

**Review by melanierogan — APPROVED** [review]
Really appreciate the clear notes on the approach

File: packages/apps/api-authorizer/src/authorization.ts [hunk 1]
```diff
@@ -12,7 +12,9 @@ type HttpVerb =
   | 'CONNECT';
 
 type AccessControlRule = (
-  jwt: Jwt,
+  jwt: Jwt & {
+    payload: (JwtPayload & { gty?: string }) | string
```
Comment (jamieparkinson) [inline packages/apps/api-authorizer/src/authorization.ts:16]: I don't think we need both checks and `gty` seems more hacky that the `@client` suffix (which is, at least, inside a meaningful claim) - can we just use the latter?

Comment (alexwlchan) [reply packages/apps/api-authorizer/src/authorization.ts]: Sure, changed.

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 Wellcome Collection
+Copyright (c) 2020-2022 Wellcome Collection
```
Comment (jamieparkinson) [inline LICENSE:3]: I could never have guessed that there is [no agreed correct way](https://softwareengineering.stackexchange.com/questions/210472/is-renewal-of-mit-license-needed-on-github-at-the-beginning-of-each-year) to do something straightforward with a FOSS license

**Review by jamieparkinson — APPROVED** [review]

File: LICENSE (hunk 2 above)
Comment (alexwlchan) [reply LICENSE:3]: Oh, haha, I just did this to trigger the pre-commit hook, but apparently it only passes modified files to prettier. 🤷‍♀️

**Pull request merged**